jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsViewImpl.java

```diff
@@ -13,157 +13,181 @@
-
-    @Inject
-    @DataField
-    public FormLabel activeTasksListLabel;
-
-    private Constants constants = Constants.INSTANCE;
-
-    @PostConstruct
-    public void init() {
-        processDefinitionIdLabel.setText(constants.Process_Definition_Id());
-        processDeploymentLabel.setText(constants.Deployment_Name());
-        processVersionLabel.setText(constants.Process_Definition_Version());
-        slaComplianceLabel.setText(constants.Process_SLA_Compliance());
-        correlationKeyLabel.setText(constants.Correlation_Key());
-        stateLabel.setText(constants.Process_Instance_State());
-        activeTasksListLabel.setText(constants.Active_Tasks());
-        currentActivitiesListLabel.setText(constants.Current_Activities());
-        parentProcessInstanceIdLabel.setText(constants.Parent_Process_Instance());
-    }
+public class ProcessInstanceDetailsViewImpl extends Composite implements ProcessInstanceDetailsPresenter.ProcessInstanceDetailsView {
 
-    @Override
-    public HTML getProcessDefinitionIdText() {
-        return processDefinitionIdText;
-    }
+    private static Binder uiBinder = GWT.create(Binder.class);
 
-    @Override
-    public HTML getCurrentActivitiesListBox() {
-        return currentActivitiesListBox;
-    }
+    private final Constants constants = Constants.INSTANCE;
 
-    @Override
-    public HTML getActiveTasksListBox() {
-        return activeTasksListBox;
-    }
+    @UiField
+    NavTabs navTabs;
+
+    @UiField
+    TabContent tabContent;
+
+    private TabListItem instanceDetailsTab;
+
+    private TabPane instanceDetailsPane;
+
+    private TabListItem processVariablesTab;
+
+    private TabPane processVariablesPane;
+
+    private TabListItem documentTab;
+
+    private TabPane documentPane;
+
+    private TabListItem logsTab;
+
+    private TabPane logsPane;
+
+    private TabListItem diagramTab;
+
+    private TabPane diagramPane;
+
+    private ProcessInstanceDetailsPresenter presenter;
 
     @Override
-    public HTML getStateText() {
-        return this.stateText;
+    public void init(final ProcessInstanceDetailsPresenter presenter) {
+        initWidget(uiBinder.createAndBindUi(this));
+        this.presenter = presenter;
+        initTabs();
     }
 
-    @Override
-    public HTML getProcessDeploymentText() {
-        return processDeploymentText;
+    public void initTabs() {
+        GWT.log("initi tabs");
+        {
+            instanceDetailsPane = GWT.create(TabPane.class);
+            instanceDetailsPane.add(presenter.getProcessInstanceView());
+
+            instanceDetailsTab = GWT.create(TabListItem.class);
+            instanceDetailsTab.setText(constants.Process_Instance_Details());
+            instanceDetailsTab.setDataTargetWidget(instanceDetailsPane);
+            instanceDetailsTab.addStyleName("uf-dropdown-tab-list-item");
+
+            tabContent.add(instanceDetailsPane);
+            navTabs.add(instanceDetailsTab);
+        }
+
```

Comment:
Maybe you don't need additional spaces for this.

Reply:
Done